test_assets/PrimvarInterpolation/README.md

```diff
@@ -0,0 +1,221 @@
+# Primvar Interpolation
+
+![annotated primvar interpolation screenshot](images/annotated_results_storm.png)
+
+USD specifies five types of interpolation for primvars in the documentation on [Interpolation of Geometric Primitive Variables](https://graphics.pixar.com/usd/release/api/class_usd_geom_primvar.html#Usd_InterpolationVals):
+
+- constant
+- uniform
+- varying
+- vertex
+- faceVarying
+
+These follow [Renderman conventions](https://renderman.pixar.com/resources/RenderMan_20/appnote.22.html#Class%20Specifiers) for primitive variables.
+
+This example asset demonstrates each primvar interpolation type. The example was inspired by / plagiarized from a [usd-interest post by Jeremy Cowles](https://groups.google.com/g/usd-interest/c/MqYLLkp-rYA). 
+
+# Mesh construction
+
+It will be helpful to review how mesh geometry is defined in USD. For this example, we define a mesh with two faces:
```

Comment:
It might be good to mention that these different interpolation modes behave slightly differently on the type of geometry - e.g. uniform means something subtly different for BasisCurve vs. Mesh

Might also be good to call out "future work" of providing a basis curves example of how primvars interpolate, I think the [USD BasisCurves doc](https://graphics.pixar.com/usd/dev/api/class_usd_geom_basis_curves.html#UsdGeomBasisCurves_PrimvarInterpolation) is pretty difficult to understand today.